src/scmdata/run.py

```diff
@@ -1824,9 +1842,10 @@ def run_append(
         ``None``.
 
     duplicate_msg
-        If "warn", raise a warning if duplicate data is detected. If "return", return
-        the joint :obj`ScmRun` (including duplicate timeseries) so the user can inspect
-        further. If ``False``, take the average and do not raise a warning.
+            If ``"warn"``, raise a warning if duplicate data is detected. If ``True``,
```

Comment:
Maybe the default behaviour should be raising `NonUniqueMetadataError`? THoughts?

Reply:
that would be my preference too, I'll change across now